rpc/coreclient.go

```diff
@@ -0,0 +1,71 @@
+package rpc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/celestiaorg/celestia-core/rpc/client/http"
+	ctypes "github.com/celestiaorg/celestia-core/rpc/core/types"
+	"github.com/celestiaorg/celestia-core/types"
+)
+
+const NewBlockSubscriber = "NewBlock/Events"
```

Comment:
nit: does this need to be exported? 
Unless we are not exposing HTTP.Subscribe in Client there would be no use cases for it.